### PR TITLE
[IMP] base: remove dead code

### PR DESCRIPTION
**Pull request by Xavier-Do**
Cleanup some code following 8cc066173dfb61bd95b8e1f0716f71f4e251810a

### CODE REVIEW

**Review by rco-odoo — COMMENTED** [review]
Wow, that looks nice!
My main comment would be the typo: usefull → useful :wink:

**Comment by kebeclibre** [comment]
@Arcasias Can you take a look ? It seems ok for me...

File: odoo/addons/base/models/ir_qweb.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def _compile_directive_call_assets(self, el, options, indent):
 
         code = self._flushText(options, indent)
         code.append(self._indent(dedent("""
-            t_call_assets_nodes = self._get_asset_nodes(%(xmlid)s, compile_options, css=%(css)s, js=%(js)s, debug=values.get("debug"), async_load=%(async_load)s, defer_load=%(defer_load)s, lazy_load=%(lazy_load)s, media=%(media)s)
+            t_call_assets_nodes = self._get_asset_nodes(%(xmlid)s, css=%(css)s, js=%(js)s, debug=values.get("debug"), async_load=%(async_load)s, defer_load=%(defer_load)s, lazy_load=%(lazy_load)s, media=%(media)s)
```
Comment (Xavier-Do) [inline odoo/addons/base/models/ir_qweb.py:195]: @Gorash can you check that

Comment (Gorash) [reply odoo/addons/base/models/ir_qweb.py:195]: Looks good to me. :+1: 
Effectively options in `_get_asset_content` were no longer useful and therefore the options parameters either.

**Review by rco-odoo — APPROVED** [review]

**Review by Arcasias — APPROVED** [review]

**Comment by Xavier-Do** [comment]
@robodoo r+

**Pull request closed without merging**